plugins/plugin-chart-table/src/transformProps.ts

```diff
@@ -88,86 +67,73 @@ const processDataRecords = memoizeOne(function processDataRecords(
   return data;
 });
 
-const isEqualColumns = <T extends TableChartProps[]>(propsA: T, propsB: T) => {
-  const a = propsA[0];
-  const b = propsB[0];
-  return (
-    a.datasource.columnFormats === b.datasource.columnFormats &&
-    a.datasource.verboseMap === b.datasource.verboseMap &&
-    a.formData.tableTimestampFormat === b.formData.tableTimestampFormat &&
-    a.formData.timeGrainSqla === b.formData.timeGrainSqla &&
-    isEqualArray(a.formData.metrics, b.formData.metrics) &&
-    isEqualArray(a.queriesData?.[0]?.data?.columns, b.queriesData?.[0]?.data?.columns)
-  );
-};
-
 const processColumns = memoizeOne(function processColumns(props: TableChartProps) {
   const {
     datasource: { columnFormats, verboseMap },
-    formData: {
-      tableTimestampFormat,
-      timeGrainSqla: granularity,
+    rawFormData: {
+      table_timestamp_format: tableTimestampFormat,
+      time_grain_sqla: granularity,
       metrics: metrics_,
-      percentMetrics: percentMetrics_,
+      percent_metrics: percentMetrics_,
```

Comment:
I'm just sick and tire of worrying about casing so I choose to use raw form data here. Should probably gradually deprecating this auto-case-transformation for the final charting API as well...